logging/dedicated-efk-logging.adoc

```diff
@@ -0,0 +1,65 @@
+:context: dedicated-efk-logging
+[id="dedicated-efk-logging"]
+= About cluster logging and {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+As an {product-title} cluster administrator, you can deploy cluster logging
+to aggregate logs for a range of services.
+
+{product-title} clusters come configured with logging via `ElasticSearch`
+operators installed and `Curator` pre-installed to have a two-day retention of
```

Comment:
Operators

Reply:
I have Elasticsearch Operator as singular. 
You might want to check with the team if they want the Cluster Logging operator mentioned. That must be installed to get the Cluster Logging CRD and CR. Maybe they want to simplify?

Reply:
Yep, singular `Operator` per discussion about Curator being a tool.